### PR TITLE
Update autodl.md

### PR DESCRIPTION
**Pull request by youngkeys**
Autodl patch for SSL: unknown error code: 5

### CODE REVIEW

File: docs/applications/autodl.md [hunk 1]
```diff
@@ -62,4 +62,13 @@ sudo systemctl disable irssi@<username>
 
 Configuration options for the GUI client can be found [here](https://code.google.com/archive/p/rutorrent/wikis/PluginAutodlirssi.wiki#Usage). The options will be found under the header "Usage".
 
-If you don't want to use the ruTorrent plugin and would prefer to configure the configuration files by hand, you can read the [autodl-irssi documentation](https://autodl-community.github.io/autodl-irssi/configuration/overview/) for an in-depth review of the available options.
\ No newline at end of file
+If you don't want to use the ruTorrent plugin and would prefer to configure the configuration files by hand, you can read the [autodl-irssi documentation](https://autodl-community.github.io/autodl-irssi/configuration/overview/) for an in-depth review of the available options.
+
+## Patch for 'Could not read: SSL: Unknown error code: 5'
```
Comment (flying-sausages) [inline docs/applications/autodl.md:67]: Could this be rewritten into the `## Troubleshooting` format with the snippet import? Might as well not lose this

**Review by flying-sausages — CHANGES_REQUESTED** [review]
Would be nice to merge this if it's updated to the new repo structure. Otherwise we should close this, it's been 6+ months this moved

**Comment by liaralabs** [comment]
Closing this since the workaround has been patched directly into autodl

File: docs/applications/autodl.md [hunk 2]
```diff
@@ -62,4 +62,13 @@ sudo systemctl disable irssi@<username>
 
 Configuration options for the GUI client can be found [here](https://code.google.com/archive/p/rutorrent/wikis/PluginAutodlirssi.wiki#Usage). The options will be found under the header "Usage".
 
-If you don't want to use the ruTorrent plugin and would prefer to configure the configuration files by hand, you can read the [autodl-irssi documentation](https://autodl-community.github.io/autodl-irssi/configuration/overview/) for an in-depth review of the available options.
\ No newline at end of file
+If you don't want to use the ruTorrent plugin and would prefer to configure the configuration files by hand, you can read the [autodl-irssi documentation](https://autodl-community.github.io/autodl-irssi/configuration/overview/) for an in-depth review of the available options.
```
Comment (flying-sausages) [inline docs/applications/autodl.md:65]: It would be nice to have this in though

Comment (liaralabs) [reply docs/applications/autodl.md:65]: This is already in the docs

**Pull request closed without merging**